tests/queries/0_stateless/02771_multidirectory_globs_storage_file.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+# Tags: no-replicated-database, no-parallel
+
+set -eu
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+# Data preparation.
+# Now we can get the user_files_path by use the table file function for trick. also we can get it by query as:
```

Comment:
```suggestion
# Now we can get the user_files_path by using the file function, we can also get it by this query:
```
I know we have this comment in few other tests, but could you please fix this typo here.